governance/xc_admin/packages/xc_admin_frontend/components/PermissionDepermissionKey.tsx

```diff
@@ -31,10 +31,12 @@ const PermissionDepermissionKey = ({
   isPermission,
   pythProgramClient,
   squads,
+  PROPOSER_SERVER_URL,
```

Comment:
why is this all caps? it's a normal variable not a constant in this context, right?

Reply:
same pretty much everywhere below actually

Reply:
I'll fix this